test/unit/resources/aws_ssm_activation_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'helper'
+require 'aws_ssm_activation'
+require 'aws-sdk-ssm'
+
+class AwsSsmActivationConstructorTest < Minitest::Test
+
+  def test_empty_params_not_ok
+    assert_raises(ArgumentError) { AwsSsmActivation.new(client_args: { stub_responses: true }) }
+  end
+
+  def test_empty_param_arg_not_ok
+    assert_raises(ArgumentError) { AwsSsmActivation.new(name: '', client_args: { stub_responses: true }) }
```

Comment:
Parameter name `name` should be `activation_id`.

Reply:
Updated in recent commit.